docs/pages/reference/cli/tsh.mdx

```diff
@@ -1140,6 +1141,7 @@ $ tsh request review [<flags>] <request-id>
 | `--approve` | `false` | `true` or `false` | Review proposes approval |
 | `--deny` | `false` | `true` or `false` | Review proposes denial |
 | `--reason` | none | String | Review reason message |
+| `--assume-start-time` | none | String | Sets time roles can be assumed by requestor (RFC3339) |
```

Comment:
I would have named this flag `--not-before` (this way it's clear that we're specifying the earliest possible time. No worries if that ship has already sailed though.

Reply:
The PR to add this flag is [here](https://github.com/gravitational/teleport/pull/35257) 
So its not fully locked in yet as 'assume-start-time' if you feel strongly about --not-before being the name

Reply:
I don't feel super strongly - up to you all if you decide to accept the feedback or keep it as-is. I'm fine either way 👍